composer.json

```diff
@@ -17,11 +17,11 @@
     "type": "library",
     "license": "Apache-2.0",
     "require": {
-        "php": ">=5.6",
+        "php": "^7.1",
         "symfony/options-resolver": "^2.7|^3.0|^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8"
+        "phpunit/phpunit": "^7.0"
```

Comment:
Why not version 8?

Reply:
Have to upgrade php on my computer

Reply:
Since we still have servers using php7.1 (www), it's safer to still test for this configuration.

Reply:
And phpunit 8 requires php 7.2